Dockerfile.production

```diff
@@ -0,0 +1,15 @@
+FROM node:lts-alpine
```

Comment:
I think maybe it is useful if this docker image is published to any registry like hub.docker.com or ghcr.io, 
but this PR is out of scope.